ci/azure-functions-dotnet-linux.yml

```diff
@@ -0,0 +1,43 @@
+name: Build and deploy .NET Core Function App to Azure functions app
+
+on:
+  push:
+    branches: [ $default-branch ]
+  
+  workflow_dispatch:
+
+env:
+  AZURE_FUNCTIONAPP_NAME: 'YOUR_FUNCTIONAPP_NAME'
+  AZURE_FUNCTIONAPP_SLOT_NAME: 'Production' # Change it according to your project
+  AZURE_FUNCTIONAPP_PACKAGE_PATH: './publish' # Change it according to your project
```

Comment:
This can just be `PACKAGE_PATH` as there does not seem to be anything specific to functions here